src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -264,6 +278,20 @@ else if(isFixed(loansPolicy)) {
     }
   }
 
+  private boolean isAlternateRenewal(RequestQueue requestQueue) {
+    Optional<Request> potentialRequest = requestQueue.getRequests().stream().skip(1).findFirst();
+    boolean isAlternateRenewal = false;
```

Comment:
Why do we need a temporary variable, rather than eagerly return true or false in the two places?

Reply:
This is just a coding style preference. I find it harder to follow the flow of control when there are multiple exits from a method.